### DISCUSSION TITLE
Flutter2 + Joomla 4 = multiplatform  or cross-plataform

### DISCUSSION THREAD

**Discussion by Stuartemk** [joomla/joomla-cms#32594] [2. Cold Features]
### Problem identified
Now with the new launch of cross-platform flutter2 is it possible that a web page made with Joomla 4 can become cross-platform? For example to create an application and put it on Google Playstore?

**Comment by Fedik** [comment]
Nowadays almost everything can become "cross-platform". 
All depend from your programming skills.

For info: joomla is a web application, that technically already "cross-platform".

**Comment by richard67** [comment]
I do not see why Flutter2 shall be integrated with Joomla4.

From what I can see it is a cross platform UI kit for creating apps.

Nobody keeps you from creating such apps which use the Joomla 4 Web Services aka JSON API to communicate with a Joomla site.

Find some documentation here: https://docs.joomla.org/Category:Joomla!_4.x_Web_Services .

**Comment by Stuartemk** [comment]
@Fedik  @richard67 
forgive my ignorance, but what you are saying is that with Joomla 4 it is possible to make Apps, unless they take all the information from the database as if they were a single system and not as if separated systems?

**Reply by richard67** [reply]
@Stuartemk Forgive me my ignorance, but what sense does your  request make for Joomla 4? Everybody complains why J4 is delayed so much, and at the same time the same people make requests like this here which would delay a release by some year. Keeping us busy with that is not really helpful. But that’s just my personal opinion, I am not speaking for Joomla.

**Reply by korenevskiy** [reply]
@Stuartemk These are completely different things. and completely different programming languages. Different virtual machines.
It's like trying to tie the fins of a submarine to a city bus.

**Reply by Stuartemk** [reply]
Joomla 4 should have been released more than a year ago, this topic was moved to the discussion area precisely because here it does not delay anything, to say that a topic in this area delays the launch, the same people make attack like this here which would delay a release by some year. But that’s just my personal opinion, I am not speaking for Joomla

**Reply by brianteeman** [reply]
@Stuartemk Anything is possible if you know how. However you clearly don't understand what flutter is and I suggest that you read more carefully about what flutter is - perhaps start at the high end overview https://developers.googleblog.com/2021/03/announcing-flutter-2.html
